src/components/DashboardFilters/TimePeriodFilter/TimePeriodFilter-container.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid/Grid';
+import { withStyles } from '@material-ui/core/styles';
+import Slider from '@material-ui/core/Slider';
+
+import { setDateToday, setDateYesterday } from '../helpers';
+import { TimePeriodStyles, useTimeFilterStyles } from './TimePeriodFilter-styles';
+
+const TimePeriodFilter = () => {
+
+  const TimePeriodSlider = withStyles(TimePeriodStyles)(Slider);
+  const timeFilterClasses = useTimeFilterStyles();
+
+  const today = setDateToday();
+  const yesterday = setDateYesterday();
+
+  const setTimeFilter = () => {
+    const today = new Date();
+    return today.getHours() + ":" + today.getMinutes();
+  }
+
+  const [timeValue, setTimeValue] = React.useState([]);
+
```

Comment:
timeValue chyba jest nie używany? sprawdź proszę

Reply:
tak jest nie używany, po prostu przygotowane jest to pod dalszą implementacje tego slidera